.github/workflows/pr_push.yml

```diff
@@ -48,6 +48,31 @@ jobs:
         - name: Build
           run: >
             cmake --build ${{github.workspace}}/build --config Release -j ${{matrix.nproc}}
+
+    CodeStyle:
```

Comment:
Add this as a requirement to the Build job.

Reply:
Done